index-merge.md

```diff
@@ -1,100 +0,0 @@
----
-title: Access Tables Using `IndexMerge`
-summary: Learn how to access tables using the `IndexMerge` query execution plan.
-aliases: ['/docs/dev/index-merge/','/docs/dev/reference/performance/index-merge/']
```

Comment:
Please add the following aliases into https://github.com/pingcap/docs/pull/3359/files#diff-7958dda55af6cba3f29e9e06ddd9d583R4:

'/docs/dev/index-merge/','/docs/dev/reference/performance/index-merge/','/tidb/dev/index-merge' @ireneontheway

Reply:
Please note: When we delete a file, in most cases, we need to add an alias to another document to redirect the old URLs. @ireneontheway @ran-huang

Reply:
To fix https://github.com/pingcap/docs-cn/pull/3201/, please help add the missing aliases to query-execution-plan.md in the docs-cn repo. @ireneontheway